pkg/clustermanager/cluster_manager.go

```diff
@@ -34,29 +33,35 @@ import (
 )
 
 const (
-	maxRetries             = 30
-	backOffPeriod          = 5 * time.Second
-	machineBackoff         = 1 * time.Second
-	machinesMinWait        = 30 * time.Minute
-	moveCAPIWait           = 15 * time.Minute
-	clusterWaitStr         = "60m"
-	ctrlPlaneWaitStr       = "60m"
-	deploymentWaitStr      = "30m"
-	ctrlPlaneInProgressStr = "1m"
-	etcdInProgressStr      = "1m"
+	maxRetries                 = 30
+	defaultBackOffPeriod       = 5 * time.Second
+	machineBackoff             = 1 * time.Second
+	defaultMachinesMinWait     = 30 * time.Minute
+	moveCAPIWait               = 15 * time.Minute
+	defaultMaxWaitPerMachine   = 10 * time.Minute
+	clusterWaitStr             = "60m"
+	DefaultControlPlaneWaitStr = "60m"
+	deploymentWaitStr          = "30m"
+	controlPlaneInProgressStr  = "1m"
+	etcdInProgressStr          = "1m"
+	DefaultEtcdWaitStr         = "60m"
 )
 
+var DefaultPerMachineWaitStr = defaultMaxWaitPerMachine.String()
```

Comment:
I would advice against making a global var public
Can we just export `defaultMaxWaitPerMachine` and make the importers call `.String()` as needed?

Reply:
Yes will do